src/components/Filters/FilterTags/FilterTags.js

```diff
@@ -57,13 +57,16 @@ class FilterTags extends Component {
     })),
     removeFilter: PropTypes.func.isRequired,
     onClose: PropTypes.func.isRequired,
-    getSourceLabel: PropTypes.func.isRequired,
   };
 
   static defaultProps = {
     filters: []
   };
 
+  static contextTypes = {
```

Comment:
Why are we passing the store as context type instead of 'connect' the selector ?